helpers/framed.js

```diff
@@ -33,9 +32,9 @@ export async function isFramed() {
 		new Promise(resolve => {
 			setTimeout(() => {
 				resolve(false);
```

Comment:
If it were fully supported, I would be tempted to try wrapping in `requestIdleCallback` if the root cause it really that the browser is too busy. However Safari is till lacking support, and our polyfill only applies a 1ms `setTimeout` to defer by a task, which would seem to be far too quick.